docs/cn/acl/user_guide.md

```diff
@@ -30,6 +30,26 @@ ACL客户端可以参考：**org.apache.rocketmq.example.simple**包下面的**A
 | defaultGroupPerm | DENY;PUB;SUB;PUB\|SUB | 默认的ConsumerGroup权限 |
 | topicPerms | topic=权限 | 各个Topic的权限 |
 | groupPerms | group=权限 | 各个ConsumerGroup的权限 |
```

Comment:
Add notes about deprecation of `topicPerms` and `groupPerms`, and their successor `resourcePerms`.